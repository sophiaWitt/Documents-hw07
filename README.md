# ITP 365 HW07 - Spring 2021 #

## Student information... ##
Name: Tommy Trojan

Email: ttrojan@usc.edu

Platform: [PC/MAC]

### Notes ###
