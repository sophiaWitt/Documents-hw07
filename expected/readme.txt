This file exists for .gitignore
